kubernetes/bankapp-ingress.yml

```
@@ -3,26 +3,29 @@ kind: Ingress
+    - megaproject.sandeeppokhrel.cloud
+    secretName: bankapp-tls-secret
+  ingressClassName: nginx
   rules:
-  - host: megaproject.trainwithshubham.com
+  - host: megaproject.sandeeppokhrel.cloud
     http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
+       paths:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)